cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst

```diff
@@ -11,6 +11,5 @@ Sources: Real-time
 
     JMS <jms>
     Kafka <kafka>
-    SQS <sqs>
```

Comment:
should be renamed to amazonsqs, since ui will just be using the name of the plugin.  sorry forgot to update it here